### ISSUE TITLE
The session of another user is opened. Authorization under someone else's login.

### ISSUE THREAD

**Issue by nitmanby** [Ylianst/MeshCentral#4961]
You've noticed that sometimes when I start using a meshcentral, I have another meshcentral user's account open. All device groups of the user under which the meshcentral was opened are fully open. Only after forcibly logging out of my account can I return to my meshcentral account.

**Comment by silversword411** [comment]
Another user is using your computer and logging in. Only way I can see that's possible.

**Comment by silversword411** [comment]
Another user is using your computer and logging in. Only way I can see that's possible.

**Comment by nitmanby** [comment]
> Another user is using your computer and logging in. Only way I can see that's possible.

No, we enter the mesh from different devices, and use google's two-factor authentication

**Comment by silversword411** [comment]
https://docs.tacticalrmm.com/mesh_integration/

**Comment by nitmanby** [comment]
> https://docs.tacticalrmm.com/mesh_integration/

I'm sorry, I don't understand how this relates to the problem described

**Comment by nitmanby** [comment]
I continue to watch the problem. I noticed that often authorization appears under a different user after I open a second window with meshcentral. Just to clarify, then I can see all the groups of this user, all devices, i.e. full account of another user.
